.ci/windows-release.yml

```diff
@@ -1,22 +1,19 @@
 steps:
-  - script: |
-      refreshenv
-      LIBCLANG_PATH=C:\Program Files\LLVM\lib
-      LLVM_CONFIG_PATH=C:\Program Files\LLVM\bin\llvm-config
-      ROARING_ARCH=x86-64-v2
-      cargo test --all
+  - script: 'refreshenv && cargo test --all'
+    env:
+      LIBCLANG_PATH: C:\Program Files\LLVM\lib
+      LLVM_CONFIG_PATH: C:\Program Files\LLVM\bin\llvm-config
+      ROARING_ARCH: x86-64-v2
     displayName: Cargo Test All
     condition:  and(succeeded(), contains(variables['Build.SourceBranch'], 'refs/tags/'), eq(variables['CI_JOB'], 'release' ))
   - script: 'cargo clean'
     displayName: Cargo Clean
     condition:  and(succeeded(), contains(variables['Build.SourceBranch'], 'refs/tags/'), eq(variables['CI_JOB'], 'release' ))
-  - script: |
-      cargo clean
```

Comment:
So no more cargo clean (different than in grin-wallet)?

Reply:
cargo clean was already here as a separate step above, but you're right that it should be consistent. Fixed.